Amazon.QLDB.Driver.Tests/QldbSessionTests.cs

```diff
@@ -181,7 +181,8 @@ public void TestExecuteAbortExceptionAbortsTransactionAndRethrowsException()
         [DataTestMethod]
```

Comment:
Please remove test TestStartTransactionGetsNewTransactionOrThrowsWhenDisposed in this file because another one already exists called: TestStartTransactionReturnsANewTransaction

Reply:
deleted.